include/cocaine/context.hpp

```diff
@@ -73,12 +78,15 @@ class context_t {
     port_mapping_t mapper;
 
 public:
-    context_t(config_t config, std::unique_ptr<logging::log_t> log);
+    context_t(config_t config, signal::handler_base_t& signal_handler, std::unique_ptr<logging::log_t> log);
```

Comment:
I still don't honor API and ABI breaking with every PR.

As we export `context_t` outside the libcocaine-core we must recompile every our service on any change.